spec/models/reservation_spec.rb

```diff
@@ -1115,8 +1114,8 @@
         @instrument.schedule_rules.destroy_all
         @instrument.schedule_rules.reload
         @instrument.update_attribute :reserve_interval, 15
-        @rule_9_to_5 = @instrument.schedule_rules.create(FactoryGirl.attributes_for(:schedule_rule, start_hour: 9, end_hour: 17))
-        @rule_5_to_7 = @instrument.schedule_rules.create(FactoryGirl.attributes_for(:schedule_rule, start_hour: 17, end_hour: 19))
+        @rule_9to5 = @instrument.schedule_rules.create(FactoryGirl.attributes_for(:schedule_rule, start_hour: 9, end_hour: 17))
```

Comment:
It's not used, but its companion `@rule_5to7` is. Assigning it feels like documentation, kind of like how we sometimes have a `let!(:something)` without referencing the `:something` in newer tests. I'm OK with leaving it until we update this section.